testApp/testApp.pro

```diff
@@ -10,7 +10,9 @@ TEMPLATE = app
 
 INCLUDEPATH += $$PWD/../telegram-qt
 
-LIBS += -lssl -lcrypto -lz
+!win32 {
```

Comment:
How would you manage libraries on windows? I think it (you curren master) is going to fail on linkage on win32.